src/pyflex/window_selector.py

```diff
@@ -911,12 +992,36 @@ def plot(self, filename=None):
         ax.spines['right'].set_color('none')
         ax.spines['left'].set_color('none')
         ax.spines['top'].set_color('none')
+        stepsize = 200
+        start, end = ax.get_xlim()
+        ax.xaxis.set_ticks(np.arange(start, end+1, stepsize))
```

Comment:
Yes, it is not good. Let me remove it first and make some tests... I think I must do that for a purpose. Can not remember this time...